templates/sabayon.json

```diff
@@ -1,16 +1,15 @@
 {
     "create_message": ["You just created a Sabayon container ",
-                       "(release=RELEASE, arch=ARCH, variant=VARIANT)\n\n",
-                       "To enable sshd, run: rc-update add sshd"],
+                       "(release=RELEASE, arch=ARCH, u=Unprivileged Container)\n\n",
```

Comment:
That part doesn't make sense. The lxc-download template can generate both privileged and unprivileged containers.